Neos.Neos/Tests/Behavior/Features/Bootstrap/MiscellaneousTrait.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Persistence\PersistenceManagerInterface;
+use Neos\Flow\ResourceManagement\ResourceManager;
+use Neos\Media\Domain\Model\Image;
+use Neos\Media\Domain\Repository\AssetRepository;
+use Neos\Utility\ObjectAccess;
+
+/**
+ * Step implementations for tests inside Neos.Neos
+ *
+* @internal only for behat tests within the Neos.Neos package
+*/
+trait MiscellaneousTrait
```

Comment:
I gave up on my fight against traits. But a "MiscellaneousTrait" really scares me

Reply:
yes but we will always have unstructured feature implementations that should NOT reside in the main feature context.
Thats why i introduced that. It should never reach multiple hundert lines and should be refactored when there are more methods who belong together.

Reply:
I totally agree to extract the functionality, but let's call it `AssetTrait` to prevent it to become the new t3lib_div :)

Reply:
Adjusted ;)